pkg/inventory/software/config_test.go

```diff
@@ -16,25 +16,25 @@ import (
 
 func TestSoftwareInventoryModuleConfig(t *testing.T) {
 	t.Run("via YAML", func(t *testing.T) {
-		cfg := configmock.NewSystemProbe(t)
+		cfg := configmock.New(t)
```

Comment:
Do we want the config from `datadog.yaml` or `system-probe.yaml` ? (`New` gives you datadog.yaml and `NewSystemProbe` give you the other).